PyHT6022/HantekFirmware/custom/device.c

```diff
@@ -74,12 +76,23 @@ BOOL set_voltage(BYTE channel, BYTE val)
     return TRUE;
 }
 
+void set_aadj() {
+	BYTE numChan = 1 + (EP2FIFOCFG & 0x01);
```

Comment:
It's probably better to make numChan a global variable, instead of relying on EP2FIFOCFG to store the right value.